include/openssl/crmferr.h

```diff
@@ -8,8 +8,11 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_CRMFERR_H
-# define HEADER_CRMFERR_H
+#ifndef OPENSSL_CRMFERR_H
+# define OPENSSL_CRMFERR_H
+# pragma once
+
+# define HEADER_CRMFERR_H  /* deprecated in version 3.0 */
```

Comment:
Remove this, as `openssl/crmferr.h` is new in 3.0

Reply:
Resolved by automatic detection.